### DISCUSSION TITLE
How to scale Up Netflix Conductor ?

### DISCUSSION THREAD

**Discussion by jegadjame** [Netflix/conductor#2876] [Q&A]
How to scale Up Netflix Conductor ? If we have multiple request for same task, How to make the request process parallelly and process quick instead of sequentially

**Comment by v1r3n** [comment]
Tasks within a workflow are processed based on the workflow definition.   Parallel tasks - like fork or dynamic fork are executed in parallel within a workflow execution.

For a given workflow definition, you can have a large of executions happening in concurrent and all these running workflow tasks will execute in parallel.  

You can spin up as many task workers and each of the worker can have multiple threads - where each thread will process one task giving you a massive parallelism.  e.g. if you have 100 instances of a task worker each with 10 threads, you can process 1000 tasks in parallel.

Hope this helps.

**Reply by jegadjame** [reply]
Thank You

**Reply by jegadjame** [reply]
Hi,
How can i enable workflow to run parellely?(If i have 1000 triggers to workflow and each workflow have 3 tasks. Currently it process all tasks from 1st trigger data and takes the next and does it sequentially other workflow. ) 

Any sample code or any configuration i should add

Thanks,
Jegadeesh
